azpubsub/contextvalidator/src/test/scala/AzPubSubAclAuthorizerTest.scala

```diff
@@ -0,0 +1,348 @@
kJBNlRua2tLSS9DZE1Bc0dBMVVkRHdRRUF3SUVzRENCbWdZRFZSMFJCSUdTTUlHUGdob3FMbVJ6ZEhNdVkyOXlaUzVoZW5WeVpTMTBaWE4wTG01bGRJSWJLaTVrYzNSekxtTnZjbVV1ZDJsdVpHOTNjeTFwYm5RdWJtVjBnaHNxTG1SemRITXVZMjl5WlM1M2FXNWtiM2R6TFhSemRDNXVaWFNDSFNvdVpITjBjeTVsTW1WMFpYTjBNaTVoZW5WeVpTMXBiblF1Ym1WMGdoZ3FMbVJ6ZEhNdWFXNTBMbUY2ZFhKbExXbHVkQzV1WlhRd2dhd0dBMVVkSHdTQnBEQ0JvVENCbnFDQm02Q0JtSVpMYUhSMGNEb3ZMMjF6WTNKc0xtMXBZM0p2YzI5bWRDNWpiMjB2Y0d0cEwyMXpZMjl5Y0M5amNtd3ZUV2xqY205emIyWjBKVEl3U1ZRbE1qQlVURk1sTWpCRFFTVXlNREl1WTNKc2hrbG9kSFJ3T2k4dlkzSnNMbTFwWTNKdmMyOW1kQzVqYjIwdmNHdHBMMjF6WTI5eWNDOWpjbXd2VFdsamNtOXpiMlowSlRJd1NWUWxNakJVVEZNbE1qQkRRU1V5TURJdVkzSnNNRTBHQTFVZElBUkdNRVF3UWdZSkt3WUJCQUdDTnlvQk1EVXdNd1lJS3dZQkJRVUhBZ0VXSjJoMGRIQTZMeTkzZDNjdWJXbGpjbTl6YjJaMExtTnZiUzl3YTJrdmJYTmpiM0p3TDJOd2N6QWZCZ05WSFNNRUdEQVdnQlNSbmp0RWJEMVhuRUozS2pUWFQ5SE1TcGNzMmpBZEJnTlZIU1VFRmpBVUJnZ3JCZ0VGQlFjREFnWUlLd1lCQlFVSEF3RXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnSUJBRTAvMGU0bmVxOHc0dE5WR3pzam5LUW1tVm5FMGlnUmtxMVNTbkk0TDJXenBKN1V0a2J1RWZmZ1BKRVpHZkt5aGRzVVVFR1VlZ0NMWkVmbWk0d2ZoV21tdWcxZ2dVOFdLbVZ5NTEzejUyY1h6YkpQM3NNYXNaZno0UkJ1YlZKdWlMUzJ1VG5EMkpObGV2cEwzdkJHR1Z2T2lCb1gyakhDbDJGNUdNV0wvZTdKaUtKOWtNenNENWpidktBNFJDQ0h6NVFmL0tnU0Z6ZU1qSHFQSnFCQmV4bk0wQ3dJWlBhaCtMYWcyb1RLZGttZlFpZnRzSjZwNnE3ZVI5ejRRMkl4U1JQbGhmRThLWkMxZExCd1NOMUgySFhkWDZ0SW1QSVBtNnd3eGd1eVJab0JzSC8xRTY3N0h1VWFKUEE0eFdOTkcwaXdOR044TnNLek5OMEdkMHN4OW56YkhsYlFEZEJLVFlIMUNDRk16TmNTdUJqaU1hOFQ3c3pTVStoeDFwRmplYWlYTFJ6d1pqdUlkeDQ1MnQ4b0RKY0VZNEI3KzZEQm9uTGs4TVlTNGZsUm5wcGZabnB3TERNcVVVZFRNNkJjRExGOXNhdlV1dzhaOG9OUm1jQjEwTlB3ZEplYzBzRWRoSDNNRThvVkFoWjZ2dHRFa2lLQmhPZXZjY2w4Umh1ekE1YnVjZFc2M0RkQVFPWDJGUzFoNG0vRGxIeU9PMFA3NG5WdTkzQUlnc0hPVGsvbFlxWnZXUVh5dmg4NHk4U1FQRm1wejBXOHduT1pIMUdXTXRXMkt2em8vU2phb2ZoU0NYa0FWa3hBN1dGU2llL2pCUlRxVzc0WWsxQTRIRHpCaml6UlZJM0FKRCtiWldjMGNFSlV0NHBOQ3pVQmY3M2crU0pKOWErUDVvbkI8L1g1MDlDZXJ0aWZpY2F0ZT4KICAgICAgPC9YNTA5RGF0YT4KICAgIDwvS2V5SW5mbz4KICA8L2RzOlNpZ25hdHVyZT4KICA8U3ViamVjdD4KICAgIDxOYW1lSUQ+c2FtcGxlLnRlc3QuYXp1cmUubmV0PC9OYW1lSUQ+CiAgICA8U3ViamVjdENvbmZpcm1hdGlvbiBNZXRob2Q9InVybjpvYXNpczpuYW1lczp0YzpTQU1MOjIuMDpjbTpiZWFyZXIiIC8+CiAgPC9TdWJqZWN0PgogIDxDb25kaXRpb25zIE5vdEJlZm9yZT0iMjAxOS0wNi0xOFQwNzowODoxMS41MThaIiBOb3RPbk9yQWZ0ZXI9IjIwNzktMDYtMTlUMDc6MDg6MTEuNTE4WiI+CiAgICA8QXVkaWVuY2VSZXN0cmljdGlvbj4KICAgICAgPEF1ZGllbmNlPnN2YzovL0F6UHViU3ViU2VydmljZVRlc3RAc2FtcGxlLWRzdHMtYXdhcmUtc2VydmljZS5jb3JlLndpbmRvd3MtdHN0Lm5ldC88L0F1ZGllbmNlPgogICAgPC9BdWRpZW5jZVJlc3RyaWN0aW9uPgogIDwvQ29uZGl0aW9ucz4KICA8QXR0cmlidXRlU3RhdGVtZW50PgogICAgPEF0dHJpYnV0ZSBOYW1lPSJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1lIj4KICAgICAgPEF0dHJpYnV0ZVZhbHVlPmF6cHVic3ViY2xpZW50LXVzZWFzdC5jb3JlLndpbmRvd3MubmV0PC9BdHRyaWJ1dGVWYWx1ZT4KICAgIDwvQXR0cmlidXRlPgogICAgPEF0dHJpYnV0ZSBOYW1lPSJodHRwOi8vc2NoZW1hcy5taWNyb3NvZnQuY29tL3dzLzIwMDgvMDYvaWRlbnRpdHkvY2xhaW1zL3JvbGUiPgogICAgICA8QXR0cmlidXRlVmFsdWU+VG9BdXRoZW50aWNhdGVBelB1YlN1YjwvQXR0cmlidXRlVmFsdWU+CiAgICA8L0F0dHJpYnV0ZT4KICA8L0F0dHJpYnV0ZVN0YXRlbWVudD4KPC9Bc3NlcnRpb24+\"" +
```

Comment:
this token should not appear in a test, put it to a file and read it.

Reply:
This token is used only once. It is ok to leave it here. Won't fix.